Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -37,6 +37,17 @@ def homebrew_extra_paths
     paths
   end
 
+  def homebrew_extra_isystem_paths
+    paths = []
+    # Add paths for GCC headers when building against glibc@2.13 because we have to use -nostdinc.
+    if deps.any? { |d| d.name == "glibc@2.13" }
+      gcc_include_dir, = Open3.capture2("#{cc} --print-file-name=include")
+      gcc_include_fixed_dir, = Open3.capture2("#{cc} --print-file-name=include-fixed")
+      paths << gcc_include_dir.strip << gcc_include_fixed_dir.strip
```

Comment:
```suggestion
      gcc_include_dir = Utils.safe_popen_read(cc, "--print-file-name=include").chomp
      gcc_include_fixed_dir = Utils.safe_popen_read(cc, "--print-file-name=include-fixed").chomp
      paths << gcc_include_dir<< gcc_include_fixed_dir
```

This is more consistent with what we do elsewhere, assuming this works.

Reply:
I had a feeling that Open3 wasn't what we normally used.  I'll test this in the Docker image before pushing to here.